### PR TITLE
User Experience Improvements

### PR DESCRIPTION
**Pull request by mrmundt**
This PR generally introduces overall user experience improvements to the practice guides. The content is 95% the same - where appropriate, I added clarifying statements or reformatted existing information for ease of use. In particular:

- "Using the Practice Guide" section added to index
- Alignment of steps with the framework image from the BSSw blog post
- Buttons to go forward and backward
- Summary on last page (with comprehensive list of resources)

### CODE REVIEW

**Comment by mrmundt** [comment]
My dev site is live: https://mrmundt.github.io/practice-guides/

File: pages/how_to_create_ptc.md [hunk 1]
```diff
@@ -1,49 +1,117 @@
...
+
+### Final Card
 
-## Step 4: Record Current PTC Values.
+| Name         | Unit Testing|
+| :-------------: | :------------- |
+| **Target**         | Apply unit testing policy to newly refactored code |
+| :-------------: | :------------- |
+| **User Story** | As a **software developer**, I want to **create a unit testing policy covering refactored code for my team** so that **I can verify that certain functions of my code are working as expected.** |
+| :-------------: | :------------- |
```
Comment (killillian) [inline pages/how_to_create_ptc.md]: This formatting row of the table, as well as line 76, show up explicitly in the page as entries in the table. I think only one row like this (i.e. line 74) is needed to align the columns, and then the others (line 76 and 78) should be removed

File: pages/how_to_create_ptc.md [hunk 2]
```diff
@@ -1,49 +1,117 @@
...
 
-Now that you have your PTC, you can Use your PTC in meetings with your team to remind everyone of your goals, and to determine whether you have met them. As seen below, you can record baseline capabilities and track progress (0-5 are suggested) for each PTC.
+| Score         | Description |
+| :-------------: | :------------- |
+| 0 | No unit testing. |
+| 1 | Unit testing implemented for one example refactored class. |
+| 2 | Two developers using unit testing process. |
+| 3 | Unit testing developer documentation completed. |
+| 4 | All developer trained to write unit tests. |
```
Comment (killillian) [inline pages/how_to_create_ptc.md]: I think this should say "developers" instead of "developer"

**Review by killillian — COMMENTED** [review]
I made two small in-line comments about typos/formatting, but overall, this looks great! I think the addition of bulleted and numbered lists in place of paragraphs of text made it much easier to read through, and the "forward" and "backward" arrows on each page made navigation simple and intuitive.

File: pages/how_to_create_ptc.md (hunk 1 above)
Comment (mrmundt) [reply pages/how_to_create_ptc.md]: I was having lots of consternation about the formatting of this page. I might just not make the Title, Target, and User Story as table elements. Maybe just a list. What do you think?

EDIT: I edited the table. Let me know if you think it's better.

**Comment by mrmundt** [comment]
Ping @elaineraybourn @wtsutherland @rmmilewi @gonsie @killillian

**Review by elaineraybourn — APPROVED** [review]
@mrmundt I like the addition of images. For now, these changes are great.

File: pages/how_to_create_ptc.md [hunk 3]
```diff
@@ -1,49 +1,112 @@
...
+
+![Step 4 of PSIP](/practice-guides/assets/images/Step4Highlight.png)
+
+Now that you have your PTC, you can Use your PTC in meetings with your team to
+remind everyone of your goals, and to determine whether you have met them.
+As seen below, you can record baseline capabilities and track progress
+for each PTC.
 
 - [x] 0 	No unit testing.
 - [ ] 1 	Unit testing implemented for one example refactored class.
```
Comment (wtsutherland) [inline pages/how_to_create_ptc.md]: Very minor thing: these checkboxes look a little cramped next to the numbers. Adding a space might make it cleaner.

**Comment by wtsutherland** [comment]
I read through everything and clicked around through the links but I could not find any significant changes. I just left one comment about a very minor spacing thing. It looks great!

**Pull request merged**